assets/js/components/ChecksResults/HintingProvider.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Why is the file named `HintingProvider`?
As far as I see, it doesn't have any relationship with the provider. It looks more some sort of placeholder

Reply:
`ResultsContainer`, `ResultsPlaceholder`, `ResultsTable`...
My prefence from left to right.
I find the component as the `content` container of this view.
But I understand it is difficult to find an obvious name

Reply:
I'll go with `ResultsContainer`, keep in mind that we have a `provider` notion in our domain and with React you can implement a pattern called Provider, where a top-layer component passes data to underlying ones. Here we adopt that logic not to pass data but to encapsulate some of the rendering logic. Anyway it has nothing to do with the cloud provider thing :joy:

Reply:
Ok, I totally missunderstood the term hehe. It could be `ResultsProvider` them as well